.azure-pipelines/linux-CI-keras-applications-nightly.yml

```diff
@@ -54,14 +54,14 @@ jobs:
         INSTALL_KERAS: pip install keras
         UNINSTALL_KERAS:
         INSTALL_TENSORFLOW: pip install tensorflow==1.15.0
-        INSTALL_ORT: pip install onnxruntime==1.1.1
+        INSTALL_ORT: pip install onnxruntime==1.3.0
         INSTALL_KERAS_RESNET: pip install keras-resnet
         INSTALL_TRANSFORMERS:
         NIGHTLY_BUILD_TEST: python run_all.py --exclude "test_keras_applications_v2.py"
 
       Python37-tf2:
-        python.version: '3.7.3'
```

Comment:
Keep it as it was?

Reply:
We have "Python37" and "Python37-official-ort" in linux nightly build, use 3.8 for this one?